card/config/initializers/recaptcha.rb

```diff
@@ -3,5 +3,5 @@
   config.public_key  = Cardio.config.recaptcha_public_key  || nil
   config.private_key = Cardio.config.recaptcha_private_key || nil
   config.proxy       = Cardio.config.recaptcha_proxy       || nil
-  config.api_version = 'v1'
+  config.api_version = 'v1' if config.respond_to?(:api_version=)
```

Comment:
This was breaking for me.  Probably because of gem versions, but this should allow it to work either way.